thrift/agent.thrift

```diff
@@ -24,6 +24,7 @@ include "jaeger.thrift"
 include "zipkincore.thrift"
 
 namespace java com.uber.jaeger.agent.thrift
+namespace netcore Jaeger.Thrift.Agent
```

Comment:
since which Thrift version is this instruction supported?

Reply:
netcore is supported for dotnet-1.0.0-preview in Thrift 0.10.0
As of today, I merged in dotnet 2.0.0 official support into the thrift master which will become 0.11.0.

Reply:
Thanks. In this case we shouldn't merge this since most of our client libs are using thrift 0.9. 

We either need to strip this string during builds of other clients, or better have it added in dotnet project only.

Reply:
@yurishkuro I ran the thrift files against 0.9.2 and didn't get an issue. I did not alter the Makefile because of the version bump. At the moment I'm using the changes that just got merged to Master by @jeking3 so can't add a build file for that unless I use an unofficial thrift docker container for the build